lib/model/app_model/list_transaction.dart

```diff
@@ -24,9 +27,14 @@ class AppListTransactions {
 class AppTransaction {
   final String txId;
 
-  AppTransaction(this.txId);
+  final String identifier;
```

Comment:
why we need it?

Reply:
This is the hash of the transaction of the payment